### ISSUE TITLE
Add a "projects using tower-lsp" section in the README

### ISSUE THREAD

**Issue by mrnossiom** [tower-lsp/tower-lsp#13]
Looking at https://github.com/ebkalderon/tower-lsp/pull/420, this is a quite good idea to show adoption and point at real examples of how the lib is used.

You can suggest projects that use tower-lsp

**Comment by timsueberkrueb** [comment]
We use `tower-lsp` for the LSP server and the [interactive web demo](https://polarity-lang.github.io/) of our [research programming language called Polarity](https://github.com/polarity-lang/polarity/).

**Comment by uncenter** [comment]
I think it might be worth clarifying more explicitly that those projects still depend on the original tower-lsp and not this fork. Took me a few minutes to realize!

**Comment by mrnossiom** [comment]
Yep, good idea. I'll also add a note that this is a fork of eyal's `tower-lsp`.

**Comment by uncenter** [comment]
Ah it looks like you accidentally changed that header in the readme to say `tower-lsp-server` in https://github.com/tower-lsp-community/tower-lsp-server/commit/54214beef2dcd8d3aa2b56341dc619acd2f97d1c, I wasn't losing my mind 😅

**Comment by timsueberkrueb** [comment]
Polarity is already using this fork :)
